subsys/console/getchar.c

```diff
@@ -13,14 +13,36 @@ static struct tty_serial console_serial;
 static u8_t console_rxbuf[CONFIG_CONSOLE_GETCHAR_BUFSIZE];
 static u8_t console_txbuf[CONFIG_CONSOLE_PUTCHAR_BUFSIZE];
 
+ssize_t console_write(void *dummy, const void *buf, size_t size)
+{
+	ARG_UNUSED(dummy);
+
+	return tty_write(&console_serial, buf, size);
+}
+
+ssize_t console_read(void *dummy, void *buf, size_t size)
+{
+	ARG_UNUSED(dummy);
+
+	return tty_read(&console_serial, buf, size);
+}
+
 int console_putchar(char c)
```

Comment:
I suggest to update the description of a function return values in a header file.

Reply:
Done